styles.css

```diff
@@ -550,6 +434,161 @@ footer .ending-line {
   margin: 0 auto 10px auto;
 }
 
-footer .footer-container li a {
-  color: #0d151e;
+@media screen and (min-width: 768px) {
+  .navbar .bars {
+    display: none;
+  }
+
+  .links-container {
+    display: flex;
+    flex-direction: row;
+    margin-right: 40px;
+  }
+
+  .links-container ul {
+    background-color: #3e3c3c;
+    display: flex;
+    flex-direction: row;
+    margin-top: 20px;
+    list-style-type: none;
+  }
+
+  .links-container ul li {
+    margin-right: 10px;
+  }
+
+  .links-container ul li a {
+    width: 45px;
+    height: 20px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    color: #fff;
+  }
+
+  .links-container .message-container {
+    margin-top: 10px;
+  }
+
+  .header .welcome {
+    padding: 25px 25px auto auto;
+    margin-left: 70px;
+  }
+
+  .header .job-desc {
+    margin-left: 70px;
+    padding: 25px 25px auto auto;
+  }
+
+  .header .intro {
+    margin-left: 70px;
+    padding: 25px 25px auto auto;
+  }
+
+  .header .icons-container {
+    position: absolute;
+    left: 0;
+    padding-top: 10%;
+  }
+
+  .header .icons-container ul {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .header .icons-container ul li {
+    margin-top: 30px;
+  }
```

Comment:
- [ ] Hi @BezzB , Great effort working on the desktop version of your portfolio project but I would kindly suggest you stick to the Figma design as much as possible, your design work cards looks different from the original design and I also noticed that while making those effort on your desktop design, your mobile design has been distrupted and its not look great like what we have in the Figma design.

Below 👇  is a video explanation of what I would suggest you really work on before you resubmit for another review